src/AddressBook/ABAddressBook.cs

```diff
@@ -207,6 +207,7 @@ public static ABAuthorizationStatus GetAuthorizationStatus ()
 		extern unsafe static void ABAddressBookRequestAccessWithCompletion (IntPtr addressbook, void * completion);
 
 		[iOS (6,0)]
+		[BindingImpl (BindingImplOptions.Optimizable)]
```

Comment:
I wonder if we can add an **intro** tests to detect them, e.g. based on any locals with a `BlockLiteral` type ?

Reply:
I already added such a test, but as an mtouch/mmp test (because intro would run on device as well, and that seemed wasteful for something that's clearly not needed): https://github.com/xamarin/xamarin-macios/pull/3391/files#diff-f773c4aa1cac370f711a23af6d86eb2aR29

Reply:
that's weird... GitHub shows me **New changes since you last viewed** that are way older than what I reviewed ?!?
anyhow I see them now and it's even better the way you implemented them!